packages/app-mobile/components/Dropdown.tsx

```diff
@@ -84,23 +116,26 @@ class Dropdown extends React.Component {
 			}
 		}
 
-		if (this.props.labelTransform && this.props.labelTransform === 'trim') headerLabel = headerLabel.trim();
+		if (this.props.labelTransform && this.props.labelTransform === 'trim') {
+			headerLabel = headerLabel.trim();
+		}
 
 		const closeList = () => {
 			this.setState({ listVisible: false });
 		};
 
-		const itemRenderer = item => {
+		const itemRenderer = (item: DropdownListItem, index: number) => {
+			const key = (item.value ?? index).toString();
 			return (
 				<TouchableOpacity
 					style={itemWrapperStyle}
-					key={item.value}
+					key={key}
 					onPress={() => {
 						closeList();
-						if (this.props.onValueChange) this.props.onValueChange(item.value);
+						if (this.props.onValueChange) this.props.onValueChange(item.value, index);
```

Comment:
Code in `ScreenHeader.tsx` expected an `index` parameter, but it was always `undefined`. As such, an `index` parameter is added here.